content/source/docs/cloud/api/plans.html.md

```diff
@@ -113,6 +113,8 @@ This temporary URL provided by the redirect has a life of **1 minute**, and shou
 
 -> **Note:** This endpoint is available for plans using Terraform 0.12 and later.
 
+-> **Note:** For the *Terraform Enterprise* - this endpoint is available from the release v202005-1 and had stability improvements implemented in v202007-1
+
```

Comment:
That starts getting into blue box overload, so let's combine the two boxes that are about _version requirements._ (Also, the grammar and formatting need some quick fixes.)

```suggestion
-> **Note:** This endpoint is available for plans using Terraform 0.12 and later. For Terraform Enterprise, this endpoint is available from v202005-1, and its stability was improved in v202007-1.

```

Reply:
Awesome, this is the way to go, and also simplifies things . Thank you Nick for your suggestion!